legacy.go

```diff
@@ -148,6 +148,7 @@ func (app *App) RegisterLegacyModules() error {
 		pfmkeeper.DefaultForwardTransferPacketTimeoutTimestamp,
 	)
 	transferStack = blockibc.NewIBCMiddleware(transferStack, app.FTFKeeper)
+	transferStack = dollar.NewIBCModule(transferStack, app.DollarKeeper)
```

Comment:
@jtieri Is this the correct ordering?

Reply:
This should be correct. 

On `RecvPacket` the calls will originate from IBC core and move towards the underlying app (in this case transfer), so after `channel.RecvPacket` the `dollar.RecvPacket` call should happen before any other middleware is invoked.